Cache/IO.lean

```diff
@@ -132,6 +132,7 @@ private def CacheM.getContext : IO CacheM.Context := do
     ("Mathlib", root),
     ("Archive", root),
     ("Counterexamples", root),
+    ("MathlibTest", root / "MathlibTest"),
```

Comment:
The other lines that use `root` do not have `/ "MathlibTest"` as well.  Is this correct as is?

Reply:
Good question! Probably removing the `MathlibTest` is more correct.
```suggestion
    ("MathlibTest", root),